tests/playwright-tests/tests/public/redirects.spec.ts

```diff
@@ -1,6 +1,12 @@
+/* eslint-disable no-restricted-syntax */
+/* eslint-disable no-await-in-loop */
```

Comment:
I'd rather not have to break rules to write a parameterised test, but Playwright lacks Jest's async `it.each` method and so I'm sticking to doing it how the docs tell you to. Unless we definitely decide to adopt Playwright, I don't think it's worth spending more time on as this isn't even a test I'd want to run in the pipeline.